docs/lineage/dagster.md

```diff
@@ -0,0 +1,105 @@
+# Dagster Integration
+
+DataHub supports integration of
+
+- Dagster Pipeline metadata
+- Job and Op run information as well as
+- Lineage information when present
+
+## Using Datahub's Dagster Sensor
+
+Dagster sensors allow us to perform some action based on some state change. Datahub's defined dagster sensor will emits metadata after every dagster pipeline run execution. This sensor is able to register both pipeline success as well as failures. For more details about Dagster sensors please refer [Sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors).
+
+## Prerequisites
+
+1. You need to create a new dagster project. See <https://docs.dagster.io/getting-started/create-new-project>.
+2. There are two ways to define Dagster definition before starting dagster UI. One using [Definitions](https://docs.dagster.io/_apidocs/definitions#dagster.Definitions) class (recommended) and second using [Repositories](https://docs.dagster.io/concepts/repositories-workspaces/repositories#repositories).
+3. Creation of new dagster project by default use Definition class to define Dagster definition.
+
+### Setup
+
+1. You need to install the required dependency.
+
+```shell
+pip install acryl-datahub[dagster]
+```
+
+2. You need to import DataHub provided sensor definition and add it in Dagster definition or dagster repository before starting dagster UI as show below: 
+**Using Definitions class:**
+
+```python
+from dagster import Definitions
+from datahub_provider.sensors.datahub_sensors import datahub_sensor
+
+defs = Definitions(
+    sensors=[datahub_sensor],
+)
+```
+
+**Using Repository decorator:**
+
+```python
+from dagster import repository
+from datahub_provider.sensors.datahub_sensors import datahub_sensor
+
+@repository
+def my_repository():
+    return [datahub_sensor]
+```
+
+3. The DataHub provided sensor internally use below configs. You can set this configs using environment variables. If not set, sensor will take default value.
+
+   **Configuration options:**
+
+   | Name                           | Default value         | Environment variable key  | Description                                                                                   |
```

Comment:
remove this name, only keep three column Environment Variable, Default Value, Description